tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/dashboard/documentdashboard.controller.js

```diff
@@ -358,13 +358,22 @@
                     downloadAttachmentsAsStream(mailAttachmentDetailsRequest, function (response) {
                         var fileName = response.fileName
                         var blob = new Blob([response.fileAttachment.result], { type: "text/eml" });
-                        //window.navigator.msSaveOrOpenBlob(blob, "temp.eml");
-                        var element = window.document.createElement("a");
-                        element.href = window.URL.createObjectURL(blob);
-                        element.download = fileName;
-                        document.body.appendChild(element);
-                        element.click();
-                        document.body.removeChild(element);
+
+
+                        if (window.navigator.msSaveOrOpenBlob) {

```

Comment:
Email as attachment was not working in IE11. to solve this issue, this code was written